jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/Classpath.java

```diff
@@ -0,0 +1,16 @@
+package com.github.victools.jsonschema.plugin.maven;
+
+public enum Classpath {
```

Comment:
**suggestion**: The name is ambiguous and too generic. How about `ClasspathFilter` or `ClasspathType` instead?

Reply:
Changed that